### ISSUE TITLE
[VT-55] [vit] Use of uninitialized value $attr in string eq at /usr/local/bin/vit 703

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#77]
_Jean-Philippe Rutault on 2013-03-15T10:19:05Z says:_ 

 OS: Archlinux 64 bits up to date
Under vit-0.7, keystroke display error message "Use of uninitialized value $attr in string eq at /usr/local/bin/vit" that mess up display.
Tested on multiple terminal emulator (gnome-terminal, xterm, guake) same result

Workaround: start vit with the following command line vit 2>/dev/null or delete -w in shebang line

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-03-15T10:19:05Z
Modified: 2014-02-09T02:23:41Z
```

**Comment by taskwarrior** [comment]
_steve rader on 2013-03-24T14:49:20Z says:_ 

 Fixed in v1.0
